### DISCUSSION TITLE
Get seo data from entries

### DISCUSSION THREAD

**Discussion by driesbonte** [ethercreative/seo#346] [Q&A] [answered]
Hi @Tam ,

Is there a way to get the seo data from an entry when you query the entry manual?

```
{% set items = craft.entries().site('international').all() %}
{% for entry in entries %}
{# get seo data form entry #}
```

Thanks

**Comment by Tam** [comment, marked as answer]
Just query it the way you would query any field (so `entry.seo` for example). If you want to set the global SEO variable based off one of the entries in the loop you can simply do `{% set seo = entry.seo %}`.
